packages/mattermost-redux/src/selectors/entities/general.ts

```diff
@@ -25,6 +25,12 @@ export function getLicense(state: GlobalState): ClientLicense {
     return state.entities.general.license;
 }
 
+export const isCloudLicense: (state: GlobalState) => boolean = createSelector(
```

Comment:
nice! we should replace all occurrence of this in a different PR to don't polute this one